source/sites/mycompanyinfrance.fr/pages/ÉconomieCollaborative/VotreSituation.js

```diff
@@ -0,0 +1,136 @@
+import { ScrollToTop } from 'Components/utils/Scroll'
+import withSitePaths from 'Components/utils/withSitePaths'
+import React, { useContext } from 'react'
+import emoji from 'react-easy-emoji'
+import { Link } from 'react-router-dom'
+import Animate from 'Ui/animate'
+import checklistSvg from './images/checklist.svg'
+import { StoreContext } from './StoreContext'
+import { BackToSelection, allTrue } from './Activité'
+import { getActivité } from './reducers'
+
+let nothingToDo = activityAnswers => a => {
```

Comment:
Remplacer `a` par `activité`
```suggestion
let nothingToDo = activityAnswers => activité => {
```

Reply:
Ou plutôt que a, parler d'`id` ou de `name`